### PR TITLE
Add govuk web banners gem

### PR DESCRIPTION
**Pull request by KludgeKML**
Add govuk_web_banners recruitment banner handling - at the moment we don't remove the existing banner code because there are incoming PRs that rely on it.

https://trello.com/c/aYXKcrCo/375-create-govukwebbanner-gem-with-support-for-recruitment-banners, [Jira issue PNP-9248](https://gov-uk.atlassian.net/browse/PNP-9248)

### CODE REVIEW

File: app/views/landing_page/blocks/_statistics.html.erb [hunk 1]
```diff
@@ -20,5 +20,4 @@
   point_size: block.data["point_size"],
   h_axis_format: block.data["x_axis_format"],
   v_axis_format: block.data["y_axis_format"],
-  hide_legend: block.data["hide_legend"]
```
Comment (hannako) [inline app/views/landing_page/blocks/_statistics.html.erb:23]: Hmm suspect this commit was intended for a different PR?!

Comment (leenagupte) [reply app/views/landing_page/blocks/_statistics.html.erb:23]: +1 This is not related to web banners.

**Review by leenagupte — COMMENTED** [review]
There's a rogue commit, otherwise it looks good.

File: app/views/landing_page/blocks/_statistics.html.erb (hunk 1 above)
Comment (KludgeKML) [reply app/views/landing_page/blocks/_statistics.html.erb]: Lol, I don't know how this got in this PR! Anyway, removed now.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -10,6 +10,7 @@
     <% unless current_page?(root_path) || !(publication || @calendar) || remove_breadcrumbs(content_item) %>
       <%= render 'govuk_publishing_components/components/contextual_breadcrumbs', content_item: content_item_hash %>
     <% end %>
+    <%= render "govuk_web_banners/recruitment_banner" %>
```
Comment (leenagupte) [inline app/views/layouts/application.html.erb:13]: The commit message about breadcrumbs got me thinking about travel advice pages. They call the same contextual breadcrumb component, so why did breadcrumbs need to be added to their show view for them to show up? They must be being removed by the `remove_breadcrumbs` method. 
Something to fix separately to this change.

**Review by leenagupte — APPROVED** [review]

**Pull request merged**